moabb/tests/datasets.py

```diff
@@ -4,6 +4,22 @@
 from moabb.datasets.fake import FakeDataset
 
 
+def _run_tests_on_dataset(d):
```

Comment:
this is more or less a duplicate of what i have in `tests/download.py`.
this require to have a local copy of the datasets. I did not find a way to skip this from travis, so i did not add it to the test suit.